tensorflow/buildspec-graviton-2-7.yml

```diff
@@ -1,39 +0,0 @@
-account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
```

Comment:
I see that graviton patch buildspecs have a naming convention that is different from the naming convention of other types of images. For eg., it has image type followed by version (buildspec-graviton-2-7.yml), whereas other images have version followed by image type (buildspec-2-0-eia.yml). In case we remove this file, we might not be able to name the buildspecs in the desired format in the future. Should we store this file, or store this info in the form of a comment?